src/Sets/BallInf.jl

```diff
@@ -337,3 +337,21 @@ function volume(B::BallInf{N}) where {N<:Real}
     vol = _vol_prod(B, n)
     return vol
 end
+
+"""
+    center(B::BallInf{N}, i::Int) where {N<:Real}
+
+Return the center along a given dimension of a ball in the infinity norm.
+
+### Input
+
+- `B` -- ball in the infinity norm
+- `i` -- dimension of interest
+
+### Output
+
+The center along a given dimension of the ball in the infinity norm.
+"""
+@inline function center(B::BallInf{N}, i::Int) where {N<:Real}
+    return B.center[i]
+end
```

Comment:
This method is not needed as it is exactly the same as the `AbstractHyperrectangle` implementation (same for `Hyperrectangle`).

Reply:
Note that the lowered code is not *exactly* the same, because of the function call. But in practice it's fine to use the the interface functions in these cases where `center` is a field (same for `Singleton` types).

```julia
julia> H = rand(Hyperrectangle, dim=100);
julia> center_v1(H, i) = H.center[i]
julia> center_v2(H, i) = center(H)[i]

julia> @btime center_v1($H, 100)
  1.304 ns (0 allocations: 0 bytes)
-0.19825521890893702

@btime center_v2($H, 100)
  1.302 ns (0 allocations: 0 bytes)
-0.19825521890893702

julia> @code_lowered center_v1(H, 10)
CodeInfo(
1 ─ %1 = Base.getproperty(H, :center)
│   %2 = Base.getindex(%1, i)
└──      return %2
)

@code_lowered center_v2(H, 10)
CodeInfo(
1 ─ %1 = Main.center(H)
│   %2 = Base.getindex(%1, i)
└──      return %2
)
```